src/lando/main/models/landing_job.py

```diff
@@ -246,6 +246,9 @@ def sort_revisions(self, revisions: list[Revision]):
                 revision=revision, landing_job=self
             ).update(index=index)
 
+    def sorted_revisions(self):
```

Comment:
Either `get_sorted_revision` or changing this into a property would be better here.